examples/click_rate_demo/thrift/click.thrift

```diff
@@ -4,11 +4,11 @@
 namespace cpp thrift
 
 enum StreamEvent {
-  CLICK,
+  CLICK = 0,
```

Comment:
don't you need some ansible stuff 
to download and install thrift. Otherwise make a readme that tells ppl what they need to run this thing.